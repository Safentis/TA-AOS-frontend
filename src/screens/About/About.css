.about__inner {
    display: grid;
    grid-template-columns: .5fr 1fr;
    grid-gap: 2rem;
    padding: 2rem;
}

.about__avatar {
    box-shadow: .3rem .3rem .5rem var(--main-black-color);
}

.about__image {
    width: 100%;
    height: 100%;
}

.about__title {
    color: var(--main-purple-color);
}

.about__text {
    color: var(--main-dark-blue-color);
    font-size: 1.6rem;
}