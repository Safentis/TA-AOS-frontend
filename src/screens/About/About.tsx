import React, { FC } from 'react';

import './About.css';
import { Props } from './About.interface';

const About: FC<Props> = ({}): React.ReactElement => {
  return (
    <section className="about">
      <div className="about__inner">
        <div className="about__avatar">
          <img className="about__image" src="https://thumbs.dfs.ivi.ru/storage5/contents/7/1/22be48dae13ba93ed78d2ccffb44fb.jpg"/>
        </div>
        <div className="about__description">
          <h2 className="about__title">
            About me
          </h2>
          <p className="about__text">
            Hello, my name is Anton i am beginner front-end developer, i like read books, play on guitar and concerts,
            my favorite books are 1983 by George Orwell and "Three friends" by Erich Maria Remarque, if talk about concerts 
            i like genre punk and post-hardcore, last concert that i visited in St. Petersburg at 2019 year, was of group the Napalm Death.
            My favorite direction is development, i like write code and to solve interesting tasks.
          </p>
          <p className="about__text">
            Thanks for a reading!
          </p>
        </div>
      </div>
    </section>
  );
};

export default About;
