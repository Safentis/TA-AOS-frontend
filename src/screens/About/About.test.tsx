import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import About from './About';

import { Props } from './About.interface';

const setUp = (props: Props) => {
  return shallow(<About {...props} />);
};

describe('<About />', () => {
  let component: ShallowWrapper<
    any,
    Readonly<{}>,
    React.Component<{}, {}, any>
  >;
  let props: Props = {};

  beforeEach(() => {
    component = setUp(props);
  });

  it('component does match to the snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
