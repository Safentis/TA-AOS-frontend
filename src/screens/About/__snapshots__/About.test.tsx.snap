// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> component does match to the snapshot 1`] = `
<section
  className="about"
>
  <div
    className="about__inner"
  >
    <div
      className="about__avatar"
    >
      <img
        className="about__image"
        src="https://thumbs.dfs.ivi.ru/storage5/contents/7/1/22be48dae13ba93ed78d2ccffb44fb.jpg"
      />
    </div>
    <div
      className="about__description"
    >
      <h2
        className="about__title"
      >
        About me
      </h2>
      <p
        className="about__text"
      >
        Hello, my name is Anton i am beginner front-end developer, i like read books, play on guitar and concerts, my favorite books are 1983 by George Orwell and "Three friends" by Erich Maria Remarque, if talk about concerts i like genre punk and post-hardcore, last concert that i visited in St. Petersburg at 2019 year, was of group the Napalm Death. My favorite direction is development, i like write code and to solve interesting tasks.
      </p>
      <p
        className="about__text"
      >
        Thanks for reading!
      </p>
    </div>
  </div>
</section>
`;
